Start.ps1

```diff
@@ -73,11 +73,13 @@ function BuildApk
 switch -wildcard (Read-Host "What would you like to do `n1.Download and Build Revamp `n2.Download and Build Xianxia `n3.Build from Source folder `n4.Build apk using CoC-AIR.swf `n5.Clean the Directory`n") 
 { 
     "1*" {
+		[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12

```

Comment:
Please add a comment for the purpose of this line.

Reply:
If I don't add this line, I'll get "Invoke-WebRequest : The request was aborted: Could not create SSL/TLS secure channel." At least TLS 1.2 is needed to access to GitHub API endpoints.